### DISCUSSION TITLE
Zones de chasse

### DISCUSSION THREAD

**Discussion by babastienne** [GeotrekCE/Geotrek-admin#4016] [Ideas]
Depuis un moment, nous entendons des territoires nous parler de l'enjeu de sensibiliser les visiteurs aux zones de chasse. Ces territoires aimeraient mettre en avant dans Geotrek, à certaines périodes de l'année, des zones de chasse sur la carte pour informer les usagers à ce sujet, un peu comme les zones de protection. Cela s'inscrit dans une démarche permettant d'éviter les conflits d'usages.

Ce ticket a pour but d'évaluer certaines possibilités quant à la fonctionnalité de mise en avant des zones de chasses.

___

Le module "Zones sensibles" dans Geotrek pourrait être modifié pour couvrir cet usage ? Quel est le besoin de gestion derrière ces informations ? Faudra-t-il ouvrir un accès au geotrek aux fédérations pour qu'elles saisissent les données ou y a t-il un référentiel de données à jour déjà existant ? 

Qu'est-ce qu'une "zone de chasse" ? Polygone avec un titre, description, date de début et de fin ?

_____

Certaines solutions existent déjà pour mettre en avant les zones de chasse, et peut-être faudrait-il les exploiter ou mettre des passerelles en place pour récupérer les données ? 
- https://www.chasseinfo.fr/ : ne possède actuellement pas d'API, le PN Forêts était intéressé pour avoir une passerelle avec cette solution.
- https://chasse-croise.net/ : la personne derrière cette solution nous a contacté et aimerait faire une passerelle avec Geotrek. La passerelle serait à double sens : récupération des infos de chasse croisé dans Geotrek pour sensibiliser les usagers et via un plugin permettre aux usagers d'ajouter une information liée à la présence de battue dans l'application chasse croisé. Pour le moment cette plateforme ne dispose pas d'API.

Y a t-il d'autres solutions développées à l'échelle nationale ? Quid de [Suricate qui avait été évoqué l'année dernière](https://www.geo.fr/animaux/suricate-en-quoi-consiste-lapp-lancee-par-le-gouvernement-pour-eviter-les-accidents-de-chasse-213161) ? 

Comment réussir à avoir une information de qualité ? Notre interlocuteur de Chassé Croisé semble dire que les chasseurs et fédérations sont réticents à partager les informations sur leur localisation à cause entre autre des conflits avec les anti-chasse.

_____

Ce sujet soulève pas mal de questions, et je pense qu'avant d'intégrer ces éléments dans Geotrek il pourrait être pertinent de former un groupe de travail avec des structures diverses pour comprendre les différents usages et besoins derrière cette problématique.

Si vous souhaitez participer à ce groupe de travail n'hésitez pas à vous manifester.

**Comment by camillemonchicourt** [comment]
Oui c'est un sujet évoqué de temps en temps.
Il y a eu des annonces du gouvernement fin 2022 sur le sujet avec l'annonce d'une application nationale où les chasseurs seraient géolocalisés.
Il a un temps été évoqué que ce serait l'application Suricate qui serait utilisée avant que cette idée ne soit finalement abandonnée il me semble : https://www.numerama.com/politique/1232508-il-ny-aura-pas-de-geolocalisation-pour-les-chasseurs-juste-une-app-de-signalement.html

Il faudrait analyser le sujet et le besoin.
Si il s'agit d'information précises et chaudes à récupérer dynamiquement, mais cela ne semble pas prêt d'exister.
Ou des infos plus générales et froides sur des zones de chasse, plus probable mais pas forcément existant ou accessible, à étudier.

Si ce sont juste des zonages, on pourrait aussi étudier la possibilité d'utiliser la table des zonages et de pouvoir publier des zonages vers l'API et donc Geotrek-rando.

**Reply by michael-albie** [reply]
L'état a mis en place une plateforme depuis Janvier 2024 en Open Data qui renseigne sur les forêts publiques, les zones chassable avec les jours de chasse, mais va englober la totalité du lot ( sera déclaré 200 ou 5000 ha chassable le même jour) n'apportant pas forcément d'information suffisamment précise pour répondre à la demande ( c'est déjà un début ).

**Reply by camillemonchicourt** [reply]
OK, je suis intéressé par plus d'infos ou un lien vers ces données.

Une autre approche pourrait de se dire que l'on n'a pas besoin de récupérer et stocker ces données dans la BDD de Geotrek-admin, mais directement les afficher sur Geotrek-rando depuis une API ou avec un widget.

**Reply by michael-albie** [reply]
Voici le lien vers les infos https://geoservices.ign.fr/services-geoplateforme-diffusion, faire votre recherche avec "Lots de Chasse"

**Reply by camillemonchicourt** [reply]
OK, j'ai regardé ces flux, mais je n'ai trouvé que les réserves nationales de chasse dans ceux-ci. 🤔

**Comment by michael-albie** [comment]
Bonjour

Le lien de la plate-forme c’est le bon, en revanche je n’ai pas contrôler
lot de chasse..!
Toutes mes excuses
